### PR TITLE
Dev th change service to orm

### PR DESCRIPTION
**Pull request by trevorhuis-smoothstack**
Hey Sean, I left the unused functions from the service and controller classes in there if you wanted to make any edits. I know this was under your responsibilities.

### CODE REVIEW

File: src/main/java/com/ss/training/lms/controller/LibrarianController.java [hunk 1]
```diff
@@ -25,139 +18,116 @@
...
+    //     }
+    //     return new ResponseEntity<List<Book>>(books, status);
+
+    // }
 
     @PutMapping(value = "/lms/librarian/branches/{branchId}/books/{bookId}")
-    public ResponseEntity<BookCopies> updateCopies(@PathVariable int branchId,
+    public ResponseEntity<String> updateCopies(@PathVariable int branchId,
                                                @PathVariable int bookId,
                                                @RequestBody BookCopies bookCopies)
```
Comment (justin-obrien-smoothstack) [inline src/main/java/com/ss/training/lms/controller/LibrarianController.java:97]: Something I didn't think of before: What if the bookCopies object has a different branchId or bookId than those given as path parameters? Currently, the path parameters aren't actually used. We could just get rid of them and use a single path for this method, or we could change the request body parameter to just an integer (the new number of copies), or we could even add the number of copies as a new path parameter, though that seems somewhat counter-intuitive.

File: src/main/java/com/ss/training/lms/controller/LibrarianController.java [hunk 2]
```diff
@@ -25,139 +18,116 @@
...
+        librarianService.updateCopies(bookCopies);
+        return new ResponseEntity<String>("success", status);
     }
 
-    @PutMapping(value = "/lms/librarian/branches/{branchId}")
+    @PutMapping(value = "/lms/librarian/branches/{branchId}/update")
     @ResponseStatus(HttpStatus.OK)
-    public ResponseEntity<LibraryBranch> updateBranch(@PathVariable int branchId,
+    public ResponseEntity<String> updateBranch(@PathVariable int branchId,
                              @RequestBody LibraryBranch libraryBranch){
```
Comment (justin-obrien-smoothstack) [inline src/main/java/com/ss/training/lms/controller/LibrarianController.java:115]: Similarly, the branchId in the path for this method isn't used either. The branch that gets updated is always the one with the ID of the one that gets passed in the request body.

**Review by justin-obrien-smoothstack — CHANGES_REQUESTED** [review]
I think when updating a branch, we should first make sure the branch exists. Similarly when changing the number of copies, we should make sure the book and branch both exist.

Also, a few of the removed methods still seem relevant. We want to be able to get a list of all branches so the user can pick what branch they manage, and a list of all books (or search by title, etc.) so they can choose what book to change the number of copies of. We also want to be able to tell them how many copies of a book their branch currently has.

**Review by sean-darsie — APPROVED** [review]
We can just leave them for now. It doesn't hurt, and we may want them later to display information to the user in a UI that we build later.

**Comment by trevorhuis-smoothstack** [comment]
I just pushed with fixes to all concerning comments

**Pull request merged**